### ISSUE TITLE
NHX is read but not render properly

### ISSUE THREAD

**Issue by smoretti** [DessimozLab/phylo-io#8]
Hi

I loaded NHX trees in phylo.io but it looks NHX tags are ignored.

Is it expected?

**Comment by Nrde** [comment]
If you mean the extra comment etc. fields then I think the answer is yes. The comment data is not displayed only the actual tree data is. If you have a small example I can have a look.

**Comment by smoretti** [comment]
Here is an example: https://selectome.unil.ch/wwwtmp/EMGT00050000004679/Drosophila/EMGT00050000004679.Drosophila.001.nhx

**Comment by smoretti** [comment]
An updated example:
https://selectome.org/euteleostomi/family/ENSGT00940000158077.1.Euteleostomi.nhx

**Comment by cdessimoz** [comment]
Indeed, the current version of phylo.io does not display the extended Newick annotations. But feel free to contribute code that addresses this! :D

**Comment by smoretti** [comment]
Duplication tag is available in the NHX tags:
`D=Y` -> duplication
`D=N` -> speciation

**Comment by alpae** [comment]
The new version available on https://beta.phylo.io is now able to properly load and render nhx files.
